src/test/java/io/dockstore/provision/DOSPluginUnitTest.java

```diff
@@ -16,23 +16,38 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 
 public class DOSPluginUnitTest {
 
     @Mock (name = "dosPluginUtil")
-    DOSPluginUtil dosPluginUtil;
-    @InjectMocks DOSPlugin.DOSPreProvision dosPreProvision;
+    private DOSPluginUtil dosPluginUtil;
+    @InjectMocks
+    private DOSPlugin.DOSPreProvision dosPreProvision;
 
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
     }
 
+    @Test
+    public void testSetConfiguration() {
+        Map<String, String> config = new HashMap<>();
+        config.put("scheme-preference", "gcs, s3, synapse");
```

Comment:
It would be good, especially since I pointed out a potential flaw in your regex in a previous review (not sure if I was right or not), if you also tested no spaces after the commas, and less importantly, more than one space after a comma. My guess is most people, or at least I, would have no spaces between the commas.